yarn.lock

```diff
@@ -2526,8 +2526,8 @@
     react-transition-group "^4.4.5"
 
 "@mui/monorepo@https://github.com/mui/material-ui.git#master":
-  version "5.10.16"
-  resolved "https://github.com/mui/material-ui.git#2abad795e4890829deb0fb40204a100b547a3873"
+  version "5.11.8"
+  resolved "https://github.com/mui/material-ui.git#c0c3d556a4d039f615188ecc5be6123f6de9156b"
```

Comment:
That's odd, I still see the `v6-alpha` in the deployment preview even though the resolved monorepo version points straight to the merge commit with this change: https://github.com/mui/material-ui/commit/c0c3d556a4d039f615188ecc5be6123f6de9156b
<img width="694" alt="Screenshot 2023-02-08 at 21 03 46" src="https://user-images.githubusercontent.com/13808724/217639011-4c6c709d-1c01-4daf-8f42-2512c60db9bb.png">

cc @LukasTy

Reply:
Hm... Interesting issue.
I've experienced the same locally, until I've deleted `.next` folder and restarted the dev server. 🤔 
Maybe it's the same in Netlify? It might cache the built resources somewhere until you forcefully remove them. 🙈

Reply:
> I've experienced the same locally, until I've deleted .next folder and restarted the dev server. 🤔

That is what I did locally too, but I didn't expect to see this problem in a preview deployment.